scenarioo-client/app/step/step.html

```diff
@@ -144,26 +144,35 @@
                     </div>
                 </div>
                 <div id="sc-step-comparison-side-by-side-view" ng-if="comparisonViewOptions[displaySideBySideViewKey]">
-                    <div class="col-lg-6">
-                        {{baseBuild.aliasName}}
-                        <div class="sc-base-screenshot clear" ng-if="!comparisonViewOptions[displayDiffScreenshotKey]">
-                            <sc-screenshot-title build="baseBuild" name="Current" alias="{{baseBuildName}}" highlights="false"></sc-screenshot-title>
-                            <sc-annotated-screenshot screen-shot-url="screenShotUrl" visibility-toggle="showScreenAnnotations"></sc-annotated-screenshot>
+                    <div class="row">
+                        <div class="col-lg-6">
+                            <sc-screenshot-title build="baseBuild" name="Current" alias="{{baseBuildName}}"
+                                                 highlights="comparisonViewOptions[displayDiffScreenshotKey]" >
+                            </sc-screenshot-title>
                         </div>
-
-                        <div class="sc-diff-screenshot clear" ng-if="comparisonViewOptions[displayDiffScreenshotKey]">
-                            <sc-screenshot-title build="baseBuild" name="Current" alias="{{baseBuildName}}" highlights="true"></sc-screenshot-title>
-                            <sc-annotated-screenshot screen-shot-url="diffScreenShotUrl" visibility-toggle="showScreenAnnotations"></sc-annotated-screenshot>
+                        <div class="col-lg-6">
+                            <sc-screenshot-title build="comparisonBuild" name="{{comparisonName}}" highlights="false"></sc-screenshot-title>
                         </div>
                     </div>
-                    <div class="col-lg-6">
-                        <div>
-                            <sc-screenshot-title build="comparisonBuild" name="{{comparisonName}}" highlights="false"></sc-screenshot-title>
-                            <div ng-if="step.diffInfo.isAdded">
-                                <p>No comparison screenshot because this is an added step.</p>
+                    <div class="row">
+                        <div class="col-lg-6">
+                            {{baseBuild.aliasName}}
```

Comment:
if the build has an alias name, I think that this should be displayed as part of the title row, or not? Before this was even in front of the rest of the title, which I think was good, if I remember correctly.